clang/test/Sema/riscv-vector-with-openmp.c

```diff
@@ -0,0 +1,12 @@
+// RUN: %clang_cc1 -triple riscv64 -target-feature +v \
+// RUN:   -disable-O0-optnone -o - -fsyntax-only -fopenmp %s -verify 
+// REQUIRES: riscv-registered-target
+// expected-no-diagnostics
+#include <riscv_vector.h>
```

Comment:
No includes of system headers are allowed here, declare the type manually.

Reply:
This is shipped by Clang not the system